test/integration/auth-methods/digest.test.js

```diff
@@ -87,6 +89,69 @@ describe('digest auth', function () {
         });
     });
 
+    describe('with interactive mode turned off', function () {
+        before(function (done) {
+            var runOptions = {
+                collection: {
+                    item: {
+                        name: 'DigestAuth',
+                        request: {
+                            url: 'https://postman-echo.com/digest-auth',
+                            auth: {
+                                type: 'digest',
+                                digest: {
+                                    algorithm: 'MD5',
+                                    username: '{{uname}}',
+                                    password: '{{pass}}',
+                                    nonInteractive: true
```

Comment:
1. The schema will have to be updated to reflect this as well.
2. Add this test for all other applicable auth modes as well.

Reply:
1. will do
2. already done (digest auth and NTLM auth only)